src/middlewares/error-logger.js

```diff
@@ -1,14 +1,17 @@
 import logger from '../lib/logger';
-
+import { RequestValidationError } from '../lib/errors';
 // NOTE: It's only stub for logger
 
 // TODO: Should write non custom error in log file.
 //       After that developer use information from log file for writing new errors class.
-const errorLogger = (error, requset, response, next) => {
+const errorLogger = (error, request, response, next) => {
+  if (error.name === RequestValidationError.name) {
+    return next();
+  }
   logger.error('↓'.repeat(error.message.length));
```

Comment:
just use logger.error directly in your app.

If you need this extra stuff in testing, you can stub a function in setup stage, otherwise it increases complexity.

Config a console logger to use colors and it's gonna be noticeable without these arrows, so remove it